cwl/workflow/split_interval_first.cwl

```diff
@@ -0,0 +1,29 @@
+cwlVersion: v1.0
```

Comment:
rename this to something not involving "first" - probably `split_interval_list.cwl`